src/FffGcodeWriter.cpp

```diff
@@ -371,6 +373,69 @@ void FffGcodeWriter::setInfillAndSkinAngles(SliceMeshStorage& mesh)
     }
 }
 
+void FffGcodeWriter::setSupportAngles(SliceDataStorage& storage)
+{
+    const Settings& mesh_group_settings = Application::getInstance().current_slice->scene.current_mesh_group->settings;
+
+    const size_t extruder_nr = mesh_group_settings.get<ExtruderTrain&>("support_infill_extruder_nr").extruder_nr;
+    const ExtruderTrain& support_infill_extruder = Application::getInstance().current_slice->scene.extruders[extruder_nr];
```

Comment:
These two lines do the following:
1. Get the support infill extruder.
2. Get the extruder number from that extruder.
3. Throw away the reference to the support infill extruder.
4. Request the extruder with that extruder number.

This can be simplified to just step 1, like so:

```suggestion
    const ExtruderTrain& support_infill_extruder = mesh_group_settings.get<ExtruderTrain&>("support_infill_extruder_nr");
```

Reply:
Good catch, thank you, fixed